pages/projects/life-journeys.js

```diff
@@ -0,0 +1,140 @@
+import { Layout } from "../../components/organisms/Layout";
+import Head from "next/head";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/router";
+import { CallToAction } from "../../components/molecules/CallToAction";
+
+function HTMList({ tag = "ul", content, listClassName, liClassName }) {
+  const parseList = (content) =>
+    content
+      .split("*") // split the string on asterisks
+      .filter((item) => item) // filter out empty strings
+      .map((item, index) => (
+        <li className={liClassName} key={index}>
+          {item.trim()}
+        </li>
+      ));
+
+  return tag === "ul" ? (
+    <ul className={listClassName}>{parseList(content)}</ul>
+  ) : (
+    <ol className={listClassName}>{parseList(content)}</ol>
+  );
+}
+
+export default function LifeJourneys(props) {
+  const { t } = useTranslation("common", "dc");
+  const { asPath } = useRouter();
+
+  return (
+    <Layout
+      locale={props.locale}
+      langUrl={asPath}
+      breadcrumbItems={[
+        { text: "Service Canada Labs", link: "/" },
+        { text: "Projects", link: "/projects" },
+      ]}
```

Comment:
These should also translate to French.

Reply:
Yeah, wasn't sure if we were doing that with these PRs or providing translation for them at a later date. Happy to add translation if that's what you'd like, but I'll wait to hear from everyone else on your latest slack message.